docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md

```diff
@@ -112,17 +105,28 @@ You can customize your network, including the virtual network, as follows:
+- Use your own existing Key Pairs. To do this, provide the name of the Key Pair, as well as the private key content and the corresponding SSH user. This information must be the same across all the regions you provision.
+
+If you use YugabyteDB Anywhere to manage SSH Key Pairs for you and you deploy multiple YugabyteDB Anywhere instances across your environment, then the AWS provider name should be unique for each instance of YugabyteDB Anywhere integrating with a given AWS account.
+
+#### Advanced
 
 You can customize the Network Time Protocol server, as follows:
 
-- Select **Use provider’s NTP server** to enable cluster nodes to connect to the AWS internal time servers. For more information, consult the AWS documentation such as [Keeping time with Amazon time sync service](https://aws.amazon.com/blogs/aws/keeping-time-with-amazon-time-sync-service/).
-- Select **Manually add NTP Servers** to provide your own NTP servers and allow the cluster nodes to connect to those NTP servers.
-- Select **Don't set up NTP** to prevent YugabyteDB Anywhere from performing any NTP configuration on the cluster nodes. For data consistency, ensure that NTP is correctly configured on your machine image.
+- Select **Use AWS's NTP server** to enable cluster nodes to connect to the AWS internal time servers. For more information, consult the AWS documentation such as [Keeping time with Amazon time sync service](https://aws.amazon.com/blogs/aws/keeping-time-with-amazon-time-sync-service/).
+- Select **Specify Custom NTP Server(s)** to provide your own NTP servers and allow the cluster nodes to connect to those NTP servers.
+- Select **Assume NTP server configured in machine image** to prevent YugabyteDB Anywhere from performing any NTP configuration on the cluster nodes. For data consistency, ensure that NTP is correctly configured on your machine image.
 
 ## Global deployment
 
-For deployment, YugabyteDB Anywhere aims to provide you with easy access to the many regions that AWS makes available globally. To that end, YugabyteDB Anywhere allows you to select which regions to which you wish to deploy and supports two different ways of configuring your setup, based on your environment: YugabyteDB Anywhere-managed configuration and self-managed configuration.
+For deployment, YugabyteDB Anywhere aims to provide you with access to the many regions that AWS makes available globally. To that end, YugabyteDB Anywhere allows you to select which regions to which you wish to deploy and supports two different ways of configuring your setup, based on your environment: YugabyteDB Anywhere-managed configuration and self-managed configuration.
 
 ### YugabyteDB Anywhere-managed configuration
```

Comment:
@Vars-07 Should we remove this section from the docs until we turn on 'Create new VPC' for AWS for the new provider?

Reply:
Yes, we should remove that for the time being.

Reply:
I agree.

Reply:
Can you be more specific. For the time being? Til when?
I see that Create new VPC is disabled in [VPC setup](https://docs.yugabyte.com/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws/#vpc-setup) section.
Doesn't the Create new VPC correspond to [Self managed configuration](http://localhost:1313/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws/#self-managed-configuration) rather than YugabyteDB Anywhere-managed configuration?

Reply:
It looks like it corresponds to YugabyteDB Anywhere-managed configuration. 
I'm making this assumption based on the section which details `custom CIDR block` being an optional field (this is only an optional field for 'Create new VPC' and is not shown for 'Existing VPC'.
And we're mentioning `YugabyteDB Anywhere generates a YugabyteDB-specific VPC in each selected region, then interconnects them` which matches the behaviour of create new vpc.